docs/DeveloperGuide.md

```diff
@@ -330,8 +332,6 @@ The 'meetings' command allows users to view all the meetings that are scheduled
 
 The UI component for this command is the `MeetingsWindow`, which is a pop-up window displaying the meetings for the current week.
 
```

Comment:
There should be a DisplayMeetingsSequenceDiagram here. I'll add it in, thanks Rahul!